test/paypal.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
 
-import { setupSDK, insertMockSDKScript } from "@paypal/sdk-client/src";
+import { setupSDK } from "@paypal/sdk-client/src";
```

Comment:
there are a few other repos that will need a similar change - https://github.com/search?q=org%3Apaypal%20insertMockSDKScript&type=code (I think) so a follow-up PR could fix the `insertMockSDKScript` implementation and revert this current PR (just FYI)